packages/kbn-optimizer/limits.yml

```diff
@@ -121,3 +121,4 @@ pageLoadAssetSize:
   controls: 34788
   expressionPie: 26338
   sharedUX: 16225
+  ux: 20784
```

Comment:
do you mind updating the one for APM?

Reply:
hmm shouldn't this happen auto? i ran the script